webpack.config.js

```diff
@@ -0,0 +1,12 @@
+const pkg = require('./package.json');
+
+module.exports = {
+    entry: {
+        'injector.lite': './src/entries/injector.lite.js',
```

Comment:
note: Michonne looks for bower_components/fa-injector/dist/injector.js for the header after this change there are four files:
injector.lite.js
injector.lite.min.js
injector.standalone.js
injector.standalone.min.js

Should we refactor Michonne to use injector.lite.js?